demo/index.html

```diff
@@ -17,7 +17,7 @@
 
   <script type="module">
     import { html, render } from 'lit-html';
-      import '../dist/index.js';
+    import '../dist/index';
```

Comment:
Does this work without the `.js`? I _thought_ you needed it in `html` files (even though you get that error saying it shouldn't be there). I think there's an eslint setting to ignore the extension in html files.

Reply:
it still works without the `.js` extension there

Reply:
added eslint setting check instead

Reply:
Oh I was totally cool with this working without the `.js`! 😄 When I tried it, I wasn't able to get it working for some reason, but if it was working for you, let's drop the `.js` and you can get rid of the eslint exception.